infrastructure/cli/pkg/providers/aws/aws.go

```diff
@@ -1,104 +1,578 @@
 package aws
 
 import (
+	"cli/pkg/console"
 	"cli/pkg/kube"
 	"context"
 	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"math/rand"
+	"os"
+	"path"
+	"text/template"
+	"time"
+
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
 	"github.com/aws/aws-sdk-go-v2/service/eks"
-	"github.com/aws/aws-sdk-go-v2/service/eks/types"
-	"log"
-	"os"
+	eksTypes "github.com/aws/aws-sdk-go-v2/service/eks/types"
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	iamTypes "github.com/aws/aws-sdk-go-v2/service/iam/types"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/mitchellh/go-homedir"
 )
 
+const RolePolicyDocument = `{
```

Comment:
We could embed this in a file. Checkout the `workspace` module for an example :)